### PR TITLE
Fix graylog version and add simcore services to monitoring network

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- [x] fix graylog version to 3.1.2-1 as 3.1.2-2 fail to start
- [x] adds webserver/storage to the monitoring network

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/deployment-agent/Makefile [hunk 1]
```diff
@@ -22,6 +22,17 @@ PUBLIC_NETWORK := $(public_network)
 endif
 export PUBLIC_NETWORK
 
+# Network that includes all services to monitor
+#  - the idea is that it shall connect osparc stack network so that e.g. cadvisor can monitor ALL the stack
```
Comment (pcrespov) [inline services/deployment-agent/Makefile]: you mean simcore stack, right?

File: services/simcore/docker-compose.deploy.yml [hunk 2]
```diff
@@ -14,6 +14,7 @@ services:
         - traefik.http.routers.webserver.middlewares=gzip_compress@file
     networks:
       - public
+      - monitored-net
```
Comment (pcrespov) [inline services/simcore/docker-compose.deploy.yml]: let us try to be consistent with names from the beginning. Either we add ``-net`` to all or not. I kind of like the minimalist approach of not but I leave the choice to you :-)

File: services/deployment-agent/Makefile (hunk 1 above)
Comment (sanderegg) [reply services/deployment-agent/Makefile]: yes corrected. copy/paste issue

File: services/simcore/docker-compose.deploy.yml (hunk 2 above)
Comment (sanderegg) [reply services/simcore/docker-compose.deploy.yml]: renamed

**Pull request merged**